### PR TITLE
Doc standalone web console removal in 2.7

### PR DESCRIPTION
**Pull request by oafischer**
https://github.com/stolostron/backlog/issues/26706

### CODE REVIEW

File: console/console_access.adoc [hunk 1]
```diff
@@ -12,15 +30,15 @@ From the {ocp} web console, you can access the {product-title} console. You can
 
 * You can also access the console from the left-hand navigation:
```
Comment (cadawson) [inline console/console_access.adoc]: We avoid using "left-hand navigation" when we can for accessibility reasons. Can we use "main navigation" or something else?

**Review by cadawson — CHANGES_REQUESTED** [review]
One comment and a few quick updates. After the comment, feel free to merge. Thanks!

File: console/console_access.adoc [hunk 2]
```diff
@@ -1,7 +1,25 @@
...
+The {product-title} web console is integrated with the {ocp} web console as a console plug-in. You can access {product-title-short} within the {ocp-short} console from the cluster switcher by selecting *All Clusters*. The cluster switcher is a drop-down menu that initially displays `local-cluster`.
+
+Select `local-cluster` when you want to use {ocp-short} console features on the cluster where you installed {product-title-short}. Select *All Clusters* when you want to use {product-title-short} features to manage your fleet of clusters.
+
+If the cluster switcher is not present, the required console plug-ins might not be enabled. For new installations, the console plug-ins are enabled by default. If you upgraded from a previous version of {product-title-short} and want to enable the plug-ins, or if you want to disable the plug-ins, see <<ocp-plug-ins,{ocp-short} console plug-ins>>:
+
+For additional options to access your console, see the following topics:
+
+* <<from-the-openshift-container-platform-web-console,From the OpenShift Container Platform web console>>
+* <<from-the-red-hat-openshift-cli,From the Red Hat OpenShift CLI>>
```
Comment (KevinFCormier) [inline console/console_access.adoc]: These are no longer correct and should be removed.

File: console/console_access.adoc [hunk 3]
```diff
@@ -1,7 +1,25 @@
...
+
+[#ocp-plug-ins]
+== {ocp-short} console plug-ins
+
+. To disable the plug-in, be sure you are in the _Administrator_ perspective in the {ocp-short} console.
+. Find *Administration* in the navigation and click *Cluster Settings*, then click the _Configuration_ tab. 
+. From the list of _Configuration resources_, click the **Console** resource with the `operator.openshift.io` API group, which contains cluster-wide configuration for the web console. 
+. Select the _Console plug-ins_ tab. Both the `acm` and `mce` plug-ins are listed. 
+. Modify plug-in status from the table. In a few moments, you are prompted to refresh the console.
```
Comment (KevinFCormier) [inline console/console_access.adoc:14]: Everything from here to the end of this file, except for the last line, is now obsolete and should be removed.

**Review by KevinFCormier — CHANGES_REQUESTED** [review]
A few more sections to be removed. Please see inline comments. Thanks.

File: console/console_access.adoc (hunk 1 above)
Comment (oafischer) [reply console/console_access.adoc]: Removed that section so I guess it solved itself. :)

**Comment by oafischer** [comment]
Made the requested changes, thanks for the review!

**Review by KevinFCormier — APPROVED** [review]
Thanks for the additional changes.

**Pull request merged**